ext/json/tests/pass001.phpt

```diff
@@ -464,7 +464,7 @@ array(14) {
     float(1.23456789E-13)
     ["E"]=>
     float(1.23456789E+34)
-    ["_empty_"]=>
+    [""]=>
```

Comment:
If you check out the code the diffs is quite a bit different around these lines so github seems to be broken. I'd review in your favorite git tool instead.